.github/workflows/release.yaml

```diff
@@ -67,3 +67,18 @@ jobs:
       image_tags: ${{ github.event.release.tag_name }}
       git_ref: ${{ github.ref }}
     secrets: inherit
+
+  cloud-provider-plguins:
+    uses: ./.github/workflows/cloud_provider_plugins_build_and_push.yaml
+    with:
+      image_tags: ${{ github.event.release.tag_name }}
+      git_ref: ${{ github.ref }}
+    secrets: inherit
+
+  caa-peerpod-ctrl-with-cloud-provider-plguins:
```

Comment:
`plugins` typo

Reply:
This was deleted in a later commit, so should be squashed for clarity

Reply:
squashed